terms.md

```diff
@@ -0,0 +1,297 @@
+**Category**: Accessibility <br>
+**Context**: The term “midget” is considered outdated and derogatory when referring to those with shorter heights or the medical condition, dwarfism. The Little People of America organization recommends using the descriptors "short stature," "little person" or "person with dwarfism." <br>
+**Link**: https://www.diversitystyleguide.com/glossary/dwarf-little-person-midget-short-stature/ <br>
+
+- crazy <br>
+**Alternative**: unexpected, unpredictable, surprising <br>
+**Category**: Accessibility <br>
+**Context**: Avoid using as these terms are commonly used informally to denote mental instability or mental illness but can be considered offensive. The repeated use perpetuates stigma and belittles mental condition that can make it more difficult for people to seek treatment.  <br>
+**Link**: https://www.pennmedicine.org/news/news-blog/2018/september/that-crazy-why-you-might-want-to-rethink-that-word-in-your-vocabulary <br>
+https://content-guide.18f.gov/our-style/inclusive-language/ <br>
+
+- insane <br>
+**Alternative**: unexpected, unpredictable, surprising <br>
+**Category**: Accessibility <br>
+**Context**: Avoid using as these terms are commonly used informally to denote mental instability or mental illness but can be considered offensive. The repeated use perpetuates stigma and belittles mental condition that can make it more difficult for people to seek treatment.  <br>
+**Link**: https://www.pennmedicine.org/news/news-blog/2018/september/that-crazy-why-you-might-want-to-rethink-that-word-in-your-vocabulary <br>
+https://content-guide.18f.gov/our-style/inclusive-language/ <br>
+
+- freak(y) <br>
+**Alternative**: unexpected, unpredictable, surprising <br>
+**Category**: Accessibility <br>
+**Context**: Avoid using as these terms are commonly used informally to denote mental instability or mental illness but can be considered offensive. The repeated use perpetuates stigma and belittles mental condition that can make it more difficult for people to seek treatment.  <br>
+**Link**: https://www.pennmedicine.org/news/news-blog/2018/september/that-crazy-why-you-might-want-to-rethink-that-word-in-your-vocabulary <br>
+https://content-guide.18f.gov/our-style/inclusive-language/ <br>
+
+- tone deaf <br>
+**Alternative**: oblivious <br>
+**Category**: Accessibility <br>
+**Context**: Be aware that "tone deaf" refers to having or showing an obtuse insensitivity or lack of perception particularly in matters of public sentiment, opinion, or taste. It may be offensive for those that have hearing loss or impairment.  <br>
+**Link**: https://www.thereadyset.co/blog/disability-justice-in-the-workplace <br>
+
+- blind spot <br>
+**Alternative**: dead spot, unseen area <br>
+**Category**: Accessibility <br>
+**Context**: Be aware that "blind spot" has a negative connotation with lack of wisdom or inability to exercise judgment. It may cause offense to those that have hearing loss or impairment. <br>
```

Comment:
```suggestion
**Context**: Be aware that "blind spot" has a negative connotation with lack of wisdom or inability to exercise judgment. It may cause offense to those that have hearing loss or impairment. <br>
```
```suggestion
**Context**: Be aware that "blind spot" has a negative connotation with lack of wisdom or inability to exercise judgment. It may cause offense to those that have loss of sight or impairment. <br>
```